galley/pkg/server/settings/args.go

```diff
@@ -165,14 +164,6 @@ func DefaultArgs() *Args {
 	}
 }
 
-func defaultExcludedResourceKinds() []string {
```

Comment:
Same comment applies here. Do not change these files.

Reply:
Discussed offline. We don't actually need these changes right now, so removing them.